python/ray/tests/test_actor_lineage_reconstruction.py

```diff
@@ -119,5 +120,43 @@ def verify3():
     wait_for_condition(lambda: verify3())
 
 
+def test_actor_reconstruction_relies_on_plasma_object(ray_start_cluster):
```

Comment:
needs comments. at a minimum, a header docstring comment that describes the high level goal of the test.